apps/web/src/pages/templates/workflow/DigestMetadata.tsx

```diff
@@ -76,6 +82,21 @@ export const DigestMetadata = ({ control, index }) => {
           );
         }}
       />
+      <Controller
+        control={control}
+        name={`steps.${index}.metadata.resend`}
```

Comment:
Is this the control that activates the mode for updating the in-app notifications? If so this name is not that clear to me. Did you had any other candidates for it?

Reply:
Not really I went with the one I had...